src/views/empty-state.html

```diff
@@ -2,13 +2,13 @@
   <div ng-if="$ctrl.config.icon" class="blank-slate-pf-icon">
     <span class="{{$ctrl.config.icon}}"></span>
   </div>
-  <h1 id="title">
+  <h4 id="empty-state-title-{{$id}}" class="h1 title">
     {{$ctrl.config.title}}
-  </h1>
-  <p id="info" ng-if="$ctrl.config.info">
+  </h4>
+  <p id="empty-state-info-{{$id}}" class="info" ng-if="$ctrl.config.info">
```

Comment:
OK, I can buy into that. How about using pf-empty-state-xxx for the classes though?